packages/@okta/vuepress-site/docs/reference/saml-hook/index.md

```diff
@@ -334,14 +334,18 @@ The `value` object is where you specify the specific operation to perform. It is
 
 Specify the location within the assertion where you want to apply your operation using a slash-delimited path, which follows JSON Patch conventions.
 
-When you perform an `add` op to add a new attribute statement, always begin with `/claims/` and follow that with the name of the new attribute that you are adding.
+When you perform an `add` op to add a new attribute statement, begin with `/claims/` and follow that with the name of the new attribute that you are adding.
 
 When you modify an existing assertions statement, begin the path with `/subject/`, `/authentication/`, `/conditions/`, or `/claims/`, depending on which part of the assertion you want to modify. You then drill down within the child elements using slash-delimited element names, for example, `/claims/array/attributeValues/1/value`. (The `/1/` in the path indicates the index of the array.)
 
 ### URI claims
 
 Okta supports URI claims with SAML assertion hooks. When you need to replace or add a URI claim, you must encode the claim name within the command per the [JavaScript Object Notation (JSON) Pointer](https://tools.ietf.org/html/rfc6901) specification. Specifically, this replaces `~` with `~0` and  `/` with `~1`.
 
+### SessionNotOnOrAfter support
+
+If you want to add `SessionNotOnOrAfter` attribute to the `<saml:AuthnStatement>` in SAML assertion, use `add` op with the path `/authentication/sessionLifetime` and session lifetime in seconds. Refer to example [here](/docs/reference/saml-hook/#sample-listing-of-json-payload-of-response). Okta will calculate `SessionNotOnOrAfter` by adding `/authentication/sessionLifetime` value to `issueInstant` and return it in SAML assertion.
```

Comment:
```suggestion
In some scenarios, your service provider may require the `SessionNotOnOrAfter` attribute for the `<saml:AuthnStatement>` in the SAML assertion, which sets the provider session time correctly. Use `add` op with the path `/authentication/sessionLifetime` and a value for session lifetime in seconds to add this attribute. See the [Sample listing of JSON payload response](/docs/reference/saml-hook/#sample-listing-of-json-payload-of-response) for an example. Okta calculates `SessionNotOnOrAfter` by adding `/authentication/sessionLifetime` value to the `issueInstant` attribute and returns it in the SAML assertion.
```

Reply:
@nakuljoshi-okta fyi, hope you don't mind a bit of a re-write. I only add a reason why I believe you would want to update this attribute in first sentence (only add if it's accurate). I then reformatted a few items for dev docs styles.

Reply:
this re-write looks better to me. committing the suggestion.